tests/cli/test_rasa_init.py

```diff
@@ -16,11 +16,32 @@ def test_init(run: Callable[..., RunResult]):
     assert os.path.exists("data/stories.md")
 
 
+def test_init_using_init_dir_option(run: Callable[..., RunResult]):
+    os.makedirs("./workspace")
+    run("init", "--no-prompt", "--quiet", "--init-dir", "./workspace")
+
+    assert os.path.exists("./workspace/actions.py")
+    assert os.path.exists("./workspace/domain.yml")
+    assert os.path.exists("./workspace/config.yml")
+    assert os.path.exists("./workspace/credentials.yml")
+    assert os.path.exists("./workspace/endpoints.yml")
+    assert os.path.exists("./workspace/models")
+    assert os.path.exists("./workspace/data/nlu.md")
+    assert os.path.exists("./workspace/data/stories.md")
+
+
+def test_not_fount_init_path(run: Callable[..., RunResult]):
```

Comment:
Can we test the error case as well with command exit, where `--no-prompt` is given and the dir defined by `--init-dir` does not exist?

Reply:
"test_not_found_init_path" is error test case, where ``--no-prompt`` is given and the dir defined by ``--init-dir`` does not exits.
And "test_init_using_init_dir_option" test case is good case of defined dir by ``--init-dir`` option.

Reply:
Yes, I meant whether a `SysExist` is raised, but it is implicitly covered by the output check